samples/sample_common/src/vaapi_allocator.cpp

```diff
@@ -247,7 +250,7 @@ mfxStatus vaapiFrameAllocator::AllocImpl(mfxFrameAllocRequest *request, mfxFrame
                                     request->Info.Width, request->Info.Height,
                                     surfaces,
                                     surfaces_num,
-                                    &attrib, 1);
+                                    &attrib[0], attrCnt);
```

Comment:
here please use ```vector::data()``` and ```vector::size()```

Reply:
Agree with @IlyaSergeyev ; lets keep "attrib" as array